src/jitlayers.h

```diff
@@ -41,16 +41,23 @@
 // However, JITLink is a relatively young library and lags behind in platform
 // and feature support (e.g. Windows, JITEventListeners for various profilers,
 // etc.). Thus, we currently only use JITLink where absolutely required, that is,
-// for Mac/aarch64.
+// for Mac/aarch64 and Linux/aarch64.
 // #define JL_FORCE_JITLINK
 
 #if defined(_COMPILER_ASAN_ENABLED_) || defined(_COMPILER_MSAN_ENABLED_) || defined(_COMPILER_TSAN_ENABLED_)
 # define HAS_SANITIZER
 #endif
 // The sanitizers don't play well with our memory manager
 
-#if defined(_OS_DARWIN_) && defined(_CPU_AARCH64_) || defined(JL_FORCE_JITLINK) || JL_LLVM_VERSION >= 150000 && defined(HAS_SANITIZER)
+#if defined(_CPU_AARCH64_) || defined(JL_FORCE_JITLINK) || JL_LLVM_VERSION >= 150000 && defined(HAS_SANITIZER)
```

Comment:
@vchuravy @pchintalapudi just to be sure: if we're using a sanitizer with LLVM 15+ you want to force JITLink?  I think we may want to remove `defined(_CPU_AARCH64_)` here and handle that below?

Reply:
Yes we want JITLink if santitizers are enabled on >= 150000, because we hit a bug in our memory manager for Rtdyld. I'm not opposed to handling that below.